packages/ast/src/encoding/types.ts

```diff
@@ -461,6 +461,14 @@ export const getDefaultTSTypeFromProtoType = (
             return t.arrayExpression([]);
         case 'cosmos.base.v1beta1.Coin':
             return t.identifier('undefined');
+        case 'tendermint.crypto.PublicKey':
+            return t.callExpression(
+                t.memberExpression(
+                    t.identifier('PublicKey'),
+                    t.identifier('fromPartial')
+                ),
+                [t.objectExpression([])]
+            )
```

Comment:
I don't think this is the right chek here. The problem occurs for all non-nullable message fields. The `PublicKey` was just one example. 

In lint 405 you need something like this (draft):


```
    if (isNonNullable && isMessage) {
        return fieldType + '.fromPartial({})'
    }

    if (isOptional) {
        return t.identifier('undefined');
    }
```

Reply:
You're right, I'll fix it that way.